src/averell/corpora.yaml

```diff
@@ -148,3 +148,16 @@
         - stanza
         - line
         - word
+- name: Stichotheque Portuguese
+  properties:
+    slug: stichopt
+    license: Unkwown
+    language: pt
+    reader: averell.readers.stichotheque
+    url: https://gitlab.com/stichotheque/stichotheque-pt/-/archive/master/stichotheque-pt-master.zip
+    size: 11.8M
```

Comment:
I changed it in the second commit, I don't know why it is not displayed on the main page of the PR

Reply:
Oh sorry, its in the other file